### DISCUSSION TITLE
CAVEBLOCK / MINEBLOCK OR WORLDEDIT SCHEMATIC

### DISCUSSION THREAD

**Discussion by RMKkamilek** [BG-Software-LLC/SuperiorSkyblock2#2286] [Approved Feature Requests] [answered]
I suggest adding the ability to support schematics created in worldedit (currently there is probably no such thing or it doesn't work very well), or a new island generation system could be added - instead of generating air around the player, a stone or other blocks of our choice would be generated

**Comment by OmerBenGera** [comment]
I don't recommend using WorldEdit schematics, use the built-in system for that.
Regarding caveblock/mineblock, not planned at this moment.

**Comment by Lemvs** [comment, marked as answer]
Duplicate of https://github.com/BG-Software-LLC/SuperiorSkyblock2/discussions/2180

@OmerBenGera you can close it.
